samples/Basic_Mdns/app/application.cpp

```diff
@@ -193,7 +192,7 @@ void parseFile(const String& name, const String& data)
 
 void test()
 {
-	debug_i("sizeof(mDNS::Finder) = %u", sizeof(mDNS::Finder));
+	debug_i("sizeof(mDNS::Finder) = %u", sizeof(mDNS::Server));
```

Comment:
May be `"sizeof(mDNS::Finder) = %u"` should be replaced with `"sizeof(mDNS::Server) = %u"` ?